### PR TITLE
python38Packages.python-redis-lock: fix build on darwin

### PR DESCRIPTION
**Pull request by marsam**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/python-redis-lock/default.nix [hunk 1]
```diff
@@ -30,7 +31,7 @@ buildPythonPackage rec {
   disabledTests = [
     # https://github.com/ionelmc/python-redis-lock/issues/86
     "test_no_overlap2"
-  ];
+  ] ++ lib.optionals stdenv.isDarwin [ "test_reset_signalizes" "test_reset_all_signalizes" ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-redis-lock/default.nix:34]:
```suggestion
  ] ++ lib.optionals stdenv.isDarwin [
    "test_reset_signalizes"
    "test_reset_all_signalizes"
  ];
```
Why do these tests fail?

**Comment by risicle** [comment]
I guess this is a fundamental problem with the darwin sandbox, but this still fails for me when both packages (3.8 and 3.9) are built in parallel, because their test redis servers are competing for the same port number. They build successfully if built one after the other. Otherwise we get

```
 # Opening Unix socket: bind: Address already in use
```

This is on macos 10.15.

File: pkgs/development/python-modules/python-redis-lock/default.nix (hunk 1 above)
Comment (marsam) [reply pkgs/development/python-modules/python-redis-lock/default.nix]: Fixed. Thanks!

**Comment by ethancedwards8** [comment]
Result of `nixpkgs-review pr 125460` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>python38Packages.python-redis-lock</li>
    <li>python39Packages.python-redis-lock</li>
  </ul>
</details>

**Pull request merged**